ui/store/institutional/institution-actions.test.js

```diff
@@ -1,6 +1,8 @@
 import sinon from 'sinon';
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
+// TODO: Remove restricted import
+// eslint-disable-next-line import/no-restricted-paths
 import MetaMaskController from '../../../app/scripts/metamask-controller';
```

Comment:
Nit/Optional - I wonder if we can skip tests.

Reply:
I'd vote no for the same reasons in the description, we want to encourage modularity and decoupling so the problem would continue in the tests as we'd be reliant on things that should be conceptually and technically unrelated.

Reply:
Resolving since I'm happy with this answer :)